readthedocs/integrations/models.py

```diff
@@ -332,6 +350,24 @@ def can_sync(self):
             return False
 
 
+class GitHubApp(Integration):
+    integration_type_id = Integration.GITHUBAPP
+    has_sync = False
+    is_remote_only = True
+
+    class Meta:
+        proxy = True
+
+    def get_absolute_url(self):
+        # TODO how to get the URL of the project GHA view?
+        return self.provider_data.get("url", "https://example.com")
+
+    @property
+    def is_active(self):
+        # TODO decide on data structure
+        return self.provider_data.get("active", True)
```

Comment:
@stsewd These will both depend on what information we get from the GHA. Any feelings on the data structure we use in the integration?

Reply:
Not sure what would be the workflow here, like if a project is connected to a GH app we create this fake integration? When the app is uninstalled from a repository, we delete the remote repository, so this could just detect if the project has connection or not (self.project.is_github_app_project)

Reply:
That sounds close, at least for a first pass. As long as we keep the integration around after GHA is connected, `is_github_app_project` seems like it should approximate whether the account is connected.

And yeah, I think the flow would be on GHA connection -- or probably at this point on new webhook events -- we ensure an integration is present. Is there a better event to wait on?

Reply:
I might take care of the integration set up in a separate PR and treat this as the display side only. In fact, @stsewd you'd be able to more easily handle the changes needed to add the integration and migration if you're looking for another small task. I don't mind trying to digest the GHA changes either though.